rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -615,33 +657,187 @@ PYBIND11_MODULE(_transport, m) {
   ;
 
   py::class_<rosbag2_py::Player>(m, "Player")
-  .def(py::init<>())
-  .def(py::init<const std::string &>())
-  .def(
-    "play",
-    py::overload_cast<const rosbag2_storage::StorageOptions &, PlayOptions &>(
-      &rosbag2_py::Player::play),
+    // Deprecated default constructor
+  .def(py::init([]()
+    {
+      PyErr_WarnEx(PyExc_DeprecationWarning, "Player() is deprecated. Use the constructor with "
+          "full configuration parameters instead.", 1);
+      return new rosbag2_py::Player();
+    }), "Deprecated: Use constructor with full options.")
```

Comment:
What is the `"Deprecated: Use constructor with full options."` string doing here; where is it going?

Reply:
Though, this is supposed to be a deprecation notice for the default `Player()` constructor (without parameters) to be visible in the docstring.

Reply:
I see